src/helpers/clipboard/clipboard_figure_state.ts

```diff
@@ -84,12 +84,20 @@ export class ClipboardFigureState implements ClipboardState {
    */
   paste(target: Zone[]) {
     const sheetId = this.getters.getActiveSheetId();
+    const size = { height: this.copiedFigure.height, width: this.copiedFigure.width };
```

Comment:
const {width ,height} = this.copiedFigure
```suggestion
    const {width, height} = this.copiedFigure
```